traffic_ops/traffic_ops_golang/test/helpers.go

```diff
@@ -26,16 +26,57 @@ import (
 	"strings"
 )
 
-// Extract the tag annotations from a struct into a string array
+// ColsFromStructByTag extracts the tag annotations from a struct into a string array.
 func ColsFromStructByTag(tagName string, thing interface{}) []string {
-	cols := []string{}
+	return ColsFromStructByTagExclude(tagName, thing, nil)
+}
+
+// InsertAtStr inserts insertMap (string to insert at -> []insert names) into cols non-destructively.
+func InsertAtStr(cols *[]string, insertMap map[string][]string) *[]string {
```

Comment:
slices are already reference types, so unless a pointer to `nil` is somehow meaningfully different than just `nil`, these `*[]string`s should probably just be `[]string`s.